### PR TITLE
added dotnet debugging info

### PR DESCRIPTION
**Pull request by scotmeiste**
Added a sample project and readme for debugging a dotnet project with tilt and kubernetes.

### CODE REVIEW

File: dotnet/readme.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+- Docker Desktop with Kubernetes enabled
+- Tilt
+- kubectl 
+- VS Code
+
+## To Run this example
+1. Run `tilt up` - this should build the sample project docker image and launch it in tilt
+1. Run `kubectl get pods` and copy the name of the pod that starts with `dotnet-api`
+1. Open `launch.json` in the `.vscode` folder and paste the name of your pod in the `pipeArgs` property
+    1. Example: `"pipeArgs": ["exec","-i","pod/dotnet-api-6746c844d4-gj6xs","--"]`
```
Comment (nicksieger) [inline dotnet/readme.md]: I think you could also put `deploy/dotnet-api` here so that you won't have to change it when a pod is restarted.

Comment (nicksieger) [inline dotnet/readme.md]:
```suggestion
```

File: dotnet/.vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+            "type": "coreclr",
+            "request": "attach",
+            "processId": 1,
+            "justMyCode": true,
+            "pipeTransport": {
+                "pipeProgram": "kubectl",
+                "pipeArgs": [
+                    "exec",
+                    "-i",
+                    "pod/dotnet-api-f66fbd56c-6vzct", // NAME of your Pod goes here
```
Comment (nicksieger) [inline dotnet/.vscode/launch.json]:
```suggestion
                    "deploy/dotnet-api",
```

File: dotnet/readme.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+            "type": "coreclr",
+            "request": "attach",
+            "processId": 1,
+            "justMyCode": true,
+            "pipeTransport": {
+                "pipeProgram": "kubectl",
+                "pipeArgs": [
+                    "exec",
+                    "-i",
+                    "pod/dotnet-api-6746c844d4-gj6xs", // NAME of your Pod goes here
```
Comment (nicksieger) [inline dotnet/readme.md]:
```suggestion
                    "deploy/dotnet-api",
```

**Review by nicksieger — APPROVED** [review]
Thanks so much for this submission! Just a few comments about using deploy/ instead of pod/, otherwise good to go.

File: dotnet/readme.md (hunk 1 above)
Comment (scotmeiste) [reply dotnet/readme.md]: Tested this out locally and works brilliantly.

File: dotnet/readme.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+## Prerequisites
+- Docker Desktop with Kubernetes enabled
+- Tilt
+- kubectl 
+- VS Code
+
+## To Run this example
+1. Run `tilt up` - this should build the sample project docker image and launch it in tilt
+1. Run `kubectl get pods` and copy the name of the pod that starts with `dotnet-api`
```
Comment (nicksieger) [inline dotnet/readme.md]: Guess we don't need this line either right?
```suggestion
```

**Pull request merged**